app.js

```diff
@@ -41,440 +52,17 @@ var influx = new Influx.InfluxDB({
-                    message.delete();
-                }
-            } else {
-                //message.channel.send(help.message + "`" + Object.keys(plugins) + "`");
-                message.channel.send("Discord's best practices for bots state that failed commands should fail silently");
-            }
-        }
-    };
-    if (message.guild.id.toString().includes(config.logserver)) {
-        embedFields = extra.fieldGenerator(message.cleanContent, "Command");
-        console.log(embedFields)
-        client.channels.get(config.logchannel).send({embed:{
-            color: colors.green,
-            author: {
-                name : message.author.username,
-                icon_url: message.author.displayAvatarURL
-            },
-            url: extra.urlGenerator(message),
-            title: "Command ran in #" + message.channel.name,
-            fields: embedFields,
-            timestamp: new Date(),
-            footer: {
-                icon_url: client.user.displayAvatarURL,
-                text: "User ID: " + message.author.id,
-            }
-        }})
-    }
-});
-
-client.on("messageDelete", (message) => {
-    embedFields = extra.fieldGenerator(message.cleanContent, "Message");
-    console.log(embedFields)
-    client.channels.get(config.logchannel).send({embed: {
-        color: colors.red,
-        author: {
-            name: message.author.username,
-            icon_url: message.author.displayAvatarURL
-        },
-        url: extra.urlGenerator(message),
-        title: "Message ID#" + message.id + " deleted in #" + message.channel.name,
-        description: "The following message was deleted:",
-        fields: embedFields,
-        timestamp: new Date(),
-        footer: {
-            icon_url: client.user.displayAvatarURL,
-            text: "User ID: " + message.author.id,
-        }
-        
-    }});
-});
-
-client.on("messageUpdate", (oldmsg, newmsg) => {
-    if (oldmsg.cleanContent !== newmsg.cleanContent) {
-        oldFields = extra.fieldGenerator(oldmsg.cleanContent, "Old message");
-        newFields = extra.fieldGenerator(newmsg.cleanContent, "New message");
-        embedFields = oldFields.concat(newFields);
-        console.log(embedFields)
-        client.channels.get(config.logchannel).send({embed: {
-            color: colors.orange,
-            author: {
-                name: newmsg.author.username,
-                icon_url: newmsg.author.displayAvatarURL
-            },
-            url: extra.urlGenerator(newmsg),
-            title: "Message ID#" + newmsg.id + " modified in #" + newmsg.channel.name,
-            description: "The following message was modified:",
-            fields: embedFields,
-            timestamp: new Date(),
-            footer: {
-                icon_url: client.user.displayAvatarURL,
-                text: "User ID: " + newmsg.author.id,
-            }
-        }})
-    }
-});
+})
 
-client.login(config.token);
+loadEvents()
+console.log(loadplugins())
```

Comment:
should this not do its own logging? logically there's no reason for this to ever return anything if loadEvents doesn't

Reply:
It should, but it was like that before I made loadEvents and I tried to match loadEvents to it but couldn't do it cleanly